src/ProductConstructionService/ProductConstructionService.Api/AuthenticationConfiguration.cs

```diff
@@ -12,31 +13,37 @@ public static class AuthenticationConfiguration
     public const string GitHubAuthenticationKey = "GitHubAuthentication";
     public const string GitHubClientIdKey = "ClientId";
     public const string GitHubClientSecretKey = "ClientSecret";
-    public const string GitHubAgentNameKey = $"{GitHubAuthenticationKey}:AgentName";
+    public const string GitHubAgentNameKey = $"{GitHubAuthenticationKey}:UserAgentProduct";
 
     // The ConfigureAuthServices we're using has a parameter that tells the service which Authentication scheme to use
     // If an endpoints path matches the AuthenticationSchemeRequestPath, it will use the authentication scheme, otherwise, it will use the
     // Application scheme. We always want to use the Authentication scheme, so we're setting the path to an empty string
     private static readonly string AuthenticationSchemeRequestPath = string.Empty;
 
-    public static void AddAuthentication(this WebApplicationBuilder builder)
+    public static void AddAuthentication(this WebApplicationBuilder builder, bool isDevelopment)
```

Comment:
This should not be called `isDevelopment`. This method should not care about what is and what is not happening in development, let alone that there is such a thing as development.

This should reflect what functionality we're turning on. E.g. `requireEndpointAuthorization` or something.

Or, just split this into another method like I did in the `AddVmrInitialization` and we'd only call that when we deem it's needed (like in development). E.g. `AddGitHubEndpointAuthorization` or similar.

Reply:
done, I also updated the `ConfigureAuthServices` method with the same approach